deploy/crds/tower.ansible.com_v1alpha1_joblaunch_cr.yaml

```diff
@@ -2,7 +2,10 @@ apiVersion: tower.ansible.com/v1alpha1
 kind: AnsibleJob
 metadata:
   name: bigjoblaunch
-  namespace: awx
 spec:
   tower_auth_secret: toweraccess
-  job_template_name: Demo Job Template
\ No newline at end of file
+  job_template_name: Demo Job Template
+  extra_vars:
+    target_clusters:
+    - aaa_cluster
```

Comment:
Can you help me understand what `target_clusters` is again?

Reply:
Hi Matt, I think this file is just an "example". I can update this change with a more generic example to showcase the usage of `extra_vars`

Reply:
Yep, lets make this more generic... my first question when seeing that was: Which cluster? An openshift cluster? acm? a tower cluster?  Probably better to have something more akin to foo/bar etc.

Reply:
Updated this file with examples from ansible doc: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#passing-variables-on-the-command-line

Added a few more to showcase float, boolean and list types.

Reply:
@matburt  Hi Matt,  Could you please review the latest changed code?  Mike has addressed all of your comments